README.md

```diff
@@ -1,14 +1,35 @@
-javabean-tester
-===============
+# javabean-tester
+
 [![Build Status](https://travis-ci.org/hazendaz/javabean-tester.svg?branch=master)](https://travis-ci.org/hazendaz/javabean-tester)
 [![Coverage Status](https://coveralls.io/repos/hazendaz/javabean-tester/badge.svg?branch=master&service=github)](https://coveralls.io/github/hazendaz/javabean-tester?branch=master)
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=hazendaz/javabean-tester)](https://dependabot.com)
 [![Maven central](https://maven-badges.herokuapp.com/maven-central/com.github.hazendaz/javabean-tester/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.hazendaz/javabean-tester)
 [![Project Stats](https://www.openhub.net/p/javabean-tester/widgets/project_thin_badge.gif)](https://www.openhub.net/p/javabean-tester)
 [![Apache 2](http://img.shields.io/badge/license-Apache%202-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
-Sites
------
+## Latest Release
```

Comment:
The hyphen-underscore line and `##` are equivalent. Switching all headers to the format of `#` for consistency.